ui/app/src/components/ActivityDashlet/ActivityDashlet.tsx

```diff
@@ -549,7 +555,14 @@ export function ActivityDashlet(props: ActivityDashletProps) {
 											/>
 										}
 									>
-										<IconButton color="primary" size="small" onClick={loadNextPage}>
+										<IconButton
+											color="primary"
+											size="small"
+											onClick={loadNextPage}
+											aria-label={formatMessage(
+												defineMessage({ defaultMessage: 'Load {limit} more', values: { limit } })
```

Comment:
What are you using defineMessage for? I'm not sure the values and the message is correct.
```suggestion
												{ defaultMessage: 'Load {limit} more' }, { limit }
```